src/views/Miscellaneous.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-menu-button slot="start" />
+        <ion-title>{{ $t("Miscellaneous") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content>
+      <main>
+        <section>
+          <ion-list>
+            <ion-list-header>{{ $t("Miscellaneous jobs") }}</ion-list-header>
+            <ion-item v-for="job in miscellaneousJobs" :key="job.jobId" @click="viewJobConfiguration(job)">
+              <ion-label>{{ job.jobName }}</ion-label>
+              <ion-badge v-if="job.runTime" color="light">{{ timeFromNow(job.runTime)}}</ion-badge>
+              <ion-icon slot="end" :icon="chevronForwardOutline" />
```

Comment:
check other pages for how this is handled. you can also check ionic documentation about ion item for how to do this using framework capabilities.

Reply:
Got it sir. We will use "detail" attribute of ion-item instead on ion-icon.